src/services/review.js

```diff
@@ -60,18 +60,29 @@ const reviewService = {
     })
   },
 
-  updateReview: async (id, currentUserId, updateData) => {
+  updateReview: async (id, currentUserId, updateData, res) => {
```

Comment:
All other services aren't receive `res`, I think this rule is appropriate for this service too.

Reply:
I've used createForbiddenError function from errorsHelper and now I'm actually not using 'res' anywhere. So, please explain why do we need to pass it as a parametr?